prisma/migrations/20231128184811_init_0/migration.sql

```diff
@@ -0,0 +1,29 @@
+-- CreateTable
+CREATE TABLE "Role" (
+    "id" SERIAL NOT NULL,
+    "name" TEXT NOT NULL,
+    "code" TEXT NOT NULL,
+
+    CONSTRAINT "Role_pkey" PRIMARY KEY ("id")
+);
+
```

Comment:
Somewhere around here you can add a sql script to populate role table. Or you can create a separate seed script if you like. If you choose the latter do not forget to include it in your readme file.